.github/workflows/lint.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
         python-version: ["3.12"]
         lintcommand:
           - "pylint -j 2 --report no datacube"
-          - "mypy datacube"
+          - "mypy datacube tests/test_utils_rio.py"
```

Comment:
Putting this here for now since no other tests pass the type check. If more files start passing the type check this can be generalized.